### PR TITLE
Fixes Brig Physician permissions / Sec perm cleanup

### PR DESCRIPTION
**Pull request by bloons3**
## About The Pull Request
Closes https://github.com/BeeStation/BeeStation-Hornet/pull/3606
Adds the Brig Physician permission to the game properly
image

Gives it to Brig Phys
Adds it to the doors they need for their job

Maps:

- [x] ~~Box Station~~ No specific medical areas
- [x] Delta Station
- [x] Donut Station
- [x] Kilo Station
- [x] Meta Station
- [x] ~~Pubby Station~~ No specific medical areas

The Brig Physician specific permission is back. It controls access to their medical supply locker, and gives them specific permission to a few doors in Brig/Security areas. The HoS, Warden, CMO, and Captain have this permission.

The Brig Physician permission can be assigned via ID console.

## Changelog
:cl:
add: Adds Brig Physician permission node (ACCESS_BRIGPHYS = 34)
tweak: Gives Brig Physician somewhat exclusive access to their medical supply locker
add: Brig Physician permission is applied to relevant areas on maps
tweak: Other minor access control changes around Brig/Brig medical areas
/:cl:

### CODE REVIEW

**Comment by bloons3** [comment]
Should be ready for merge, leave feedback on GH

File: code/modules/jobs/job_types/chief_medical_officer.dm [hunk 1]
```diff
@@ -21,10 +21,10 @@
 
 	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_GENETICS, ACCESS_CLONING, ACCESS_APOTHECARY, ACCESS_HEADS, ACCESS_MINERAL_STOREROOM,
 			ACCESS_CHEMISTRY, ACCESS_VIROLOGY, ACCESS_CMO, ACCESS_SURGERY, ACCESS_RC_ANNOUNCE, ACCESS_MECH_MEDICAL,
-			ACCESS_KEYCARD_AUTH, ACCESS_SEC_DOORS, ACCESS_MAINT_TUNNELS)
+			ACCESS_KEYCARD_AUTH, ACCESS_SEC_DOORS, ACCESS_MAINT_TUNNELS, ACCESS_BRIGPHYS)
 	minimal_access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_GENETICS, ACCESS_CLONING, ACCESS_APOTHECARY, ACCESS_HEADS, ACCESS_MINERAL_STOREROOM,
 			ACCESS_CHEMISTRY, ACCESS_VIROLOGY, ACCESS_CMO, ACCESS_SURGERY, ACCESS_RC_ANNOUNCE, ACCESS_MECH_MEDICAL,
-			ACCESS_KEYCARD_AUTH, ACCESS_SEC_DOORS, ACCESS_MAINT_TUNNELS)
+			ACCESS_KEYCARD_AUTH, ACCESS_SEC_DOORS, ACCESS_MAINT_TUNNELS,ACCESS_BRIGPHYS)
```
Comment (utilbird) [inline code/modules/jobs/job_types/chief_medical_officer.dm]: day ruined

Comment (bloons3) [reply code/modules/jobs/job_types/chief_medical_officer.dm]: Put a space in the suggestion so I can commit it while laying in bed

Comment (park66665) [reply code/modules/jobs/job_types/chief_medical_officer.dm]:
```suggestion
			ACCESS_KEYCARD_AUTH, ACCESS_SEC_DOORS, ACCESS_MAINT_TUNNELS, ACCESS_BRIGPHYS)
```

**Review by Crossedfall — DISMISSED** [review]
Tested on meta during TM and I didn't have access to get to surgery. Should double check all of the map changes.

**Review by Crossedfall — APPROVED** [review]
No issues raised during the TM on several rounds. The map changes seem to be fine and Bloons promised that they tested it.

**Pull request merged**